imcsdk/imccoreutils.py

```diff
@@ -751,6 +752,18 @@ def validate_property_value(mo, prop, value):
     return False
 
 
+def validate_property_access(mo, prop, platform=None):
+    access_list = []
+    for platform in IMC_PLATFORM_LIST:
+        prop_ = _get_property_from_prop_meta_for_platform(
+                mo,
+                prop,
+                platform)
+        if prop_:
+            access_list.append(prop_.access)
+    return MoPropertyMeta.READ_WRITE in access_list
```

Comment:
the list is not required.. just return True in the loop and false outside of it.